packages/adapter-vercel/README.md

```diff
@@ -14,7 +14,13 @@ const vercel = require('@sveltejs/adapter-vercel');
 module.exports = {
 	kit: {
 		...
-		adapter: vercel()
+		adapter: vercel()	// outputs to `.vercel_build_output`
```

Comment:
Rather than putting this here I'd put it in the next example to keep all the output stuff next to each other:
```
vercel({ out: 'my-output-directory' }); // defaults to `.vercel_build_output`
```